_pages/dev/connecting-to-other-systems.md

```diff
@@ -1,74 +1,70 @@
+![component data binding]({{ site.baseurl }}/assets/images/data-binding-architecture.png)
 
-While this is a pretty complex setup, you don't need to worry about most layers. When you want to connect a UI component to an alternative data source, you would customize some low level layers without being concerned with the facade layer or data store. Only if alternative client-side business logic is required, you would provide additional logic, most likely close to the UI layer.
+Although this is a fairly complex setup, you do not need to worry about most layers. When you want to connect a UI component to an alternative data source, you can customize some low-level layers without being concerned with the facade layer or the data store. Only if alternative client-side business logic is required would you then provide additional logic (most likely, close to the UI layer).
 
 ## Connector Logic
 
-The Connector logic sits between the in-memory data store (ngrx) and the back end. A specific connector per domain is used to offload the connection to a back end system. For example, the product connector takes care of loading product details.
+The connector logic sits between the in-memory data store and the back end. A specific connector is used for each domain to offload the connection to a back-end system. For example, the product connector takes care of loading the product details.
 
-To provide optimal flexibility, there are three entities involved to connect a back end system:
+To provide optimal flexibility, there are three entities involved in connecting to a back-end system: a connector, an adapter, and a converter.
 
-1. Connector
-2. Adapter
-3. Convertor
+This is a common pattern across different frameworks and technology stacks, although different names are used (for example, populator or serializer instead of convertor).
 
-This is a common pattern across different frameworks and technology stacks, although different names are used (for example, convertor vs. populator vs. serializer, etc.).
+A fine-grained setup helps to separate concerns, and simplifies further customization. That being said, when you bind to an alternative data source, nothing stops you from further simplifying your setup.
 
-The fine-grained setup will help to separate concerns and simplifies further customization. That being said, when you bind to an alternative data source, nothing stops you from simplifying this setup.
+### Connector
 
-### 1. Connector
+The connector orchestrates the connection to a source system. The connector layer could be considered over-engineered, because there are occasions where standard data is provided, even in the case of switching to an alternative system. An example of this is when structured CMS data is loaded: Spartacus can be set up to add static CMS data without relying on a back end at all, or as a fall-back in case the CMS does not provide sufficient data.
```

Comment:
> An example of this is 

A real example use-case of the connector is

Reply:
Done.